### ISSUE TITLE
Using serverless-pseudo-parameters with last serverless fwk causes CloudFormation template malformed

### ISSUE THREAD

**Issue by Bond74** [svdgraaf/serverless-pseudo-parameters#45]
The plugin doesn't work properly with last serverless versions, checked with version 1.48.4.
The noticed issue is: 
 - The compiled serverless template (\.serverless\serverless-state.json) contains values like "#{AWS" which causes further CloudFormation stack creation issue

Steps to reproduce:
1) Unpack the attached file into a separate folder
 Note: below is the plugin values declaration that causes issue (see serverless.yml):
```
functions:
  test:
    handler: handler.test
    events:
      - sns:
          displayName: dmc-cnst-consenthub-preferences-changed-${opt:stage}
          arn: arn:aws:sns:#{AWS::Region}:#{AWS::AccountId}:dmc-cnst-consenthub-preferences-changed-${opt:stage}
```
2) Execute 'npm install'
3) Execute 'sls package -s dev' command and check created serverless-state.json in .serverless folder. You may see entry like "Region": "#{AWS" there - it is not correct

[example.zip](https://github.com/svdgraaf/serverless-pseudo-parameters/files/3840428/mdlwr-config-test.zip)

**Comment by svdgraaf** [comment]
Please upgrade SLS. This was a degredation in serverless framework, which was consequently fixed in 1.57.x, see #38 for more details.

**Comment by Bond74** [comment]
Checked with  sls v1.57.0 The issue is not reproducible there indeed. Thanks a lot!
